src/components/publishing/image_collection_item.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from "react"
+import styled from "styled-components"
+
+interface FillwidthItemProps extends React.HTMLProps<HTMLDivElement> {
+  width?: number
+  height?: number
+  margin?: number
+  key: string
+}
+
+const FillwidthItem: React.SFC<FillwidthItemProps> = props => {
+  const { key, ...rest } = Object.assign({}, props)
+  return <div {...rest} key={key} />
```

Comment:
Ah gotcha, yeah that makes sense 👍 

if you want typings it gets a bit hairy, something like the following should do it:

```ts
import * as React from "react"
import styled, { StyledFunction } from "styled-components"

interface FillwidthItemProps {
  width?: number
  height?: number
  margin?: number
  key: string
}

/**
 * Need to type the function first so that props are typed correctly. Alas `styled.div<P>` doesn’t work at the moment
 * due TypeScript not (yet) supporting generic on template string functions.
 * 
 * See:
 * - https://github.com/styled-components/styled-components/issues/630 
 * - https://github.com/Microsoft/TypeScript/issues/11947
 */
const div: StyledFunction<FillwidthItemProps & React.HTMLProps<HTMLDivElement>> = styled.div

export default div`
  margin-right: ${props => props.margin + "px" || 0};
  width: ${props => props.width + "px" || "auto"};
  height: ${props => props.height + "px" || "auto"};
`
```